specificatie/gba-genereervariant/openapi.json

```diff
@@ -28,7 +28,7 @@
         "operationId" : "GetPersonen",
         "requestBody" : {
           "content" : {
-            "application/json" : {
```

Comment:
@MelvLee waarom is dit nodig? betekent dit dat een gebruiker verplicht ook charset moet toevoegen aan content-type?

Reply:
Ik dacht dat je expliciet wilt opnemen in de OAS specs. Maar als je het echt goed wilt doen, dan zou je het volgens mij als volgt moeten specificeren:

```
          'application/json':
            schema:
              $ref: '#/components/schemas/PersonenQuery'
          'application/json; charset=utf-8':
            schema:
              $ref: '#/components/schemas/PersonenQuery'
```
`application/json; charset=utf-8` als content type betekent ook niet dat als in de Accept header geen `application/json; charset=utf-8' staat dat je dan een 415 krijgt.

Reply:
Eigenlijk is dit wat je wilt omdat de API zoeken op diakrieten ondersteunt. Dat moet de charset wel utf-8 zijn. Ik kan dit ook documenteren in de features overzicht.